### ISSUE TITLE
AttributeError: 'PaymentRequest' object has no attribute 'outputs'

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1503]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "_ctypes/callbacks.c", line 234, in 'calling callback function'
  File "/var/containers/Bundle/Application/FD5763F4-19C9-4042-9C2F-47BD773810A8/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app_packages/rubicon/objc/runtime.py", line 1065, in _objc_method
    result = f(py_self, *args)
  File "/private/var/containers/Bundle/Application/FD5763F4-19C9-4042-9C2F-47BD773810A8/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app/ElectronCash/electroncash_gui/ios_native/send.py", line 583, in getPayToAddress
    return ns_from_py(pr.get_address())
  File "/private/var/containers/Bundle/Application/FD5763F4-19C9-4042-9C2F-47BD773810A8/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app/ElectronCash/electroncash/paymentrequest.py", line 234, in get_address
    o = self.outputs[0]

AttributeError: 'PaymentRequest' object has no attribute 'outputs'
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.7 (iOS Build: 100)  | 3.6.6+ (default, Jul  5 2018, 10:38:52)  [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] | Darwin-18.6.0-iPhone7,2-64bit iOS 12.3.1 iPhone (A35719AF-A9BD-4128-A45A-DB1FB1D3F5DB) | standard | en_UK | 

Additional Information
------------------------

> @mhsmith here. Crash happened when scanning a QR code for an expired BitPay URL.

---

**Comment by mhsmith** [comment]
Here is the URL: bitcoincash:?r=https://bitpay.com/i/XDuMg51dXq1rroQEmtN2o7

**Comment by mhsmith** [comment]
QR code generated on https://www.the-qrcode-generator.com/, if it makes a difference.

**Comment by cculianu** [comment]
Oh wow thanks for this. Will look into it. Likely the code path isn't dealing with the error situation as expected.. or something. Thanks for all the info!

**Comment by cculianu** [comment]
Hmm, I'm having trouble reproducing it? Do I need to open the URL and/or scan the QR to do it? It just gives me the error about being expired if I open the URL.

If I scan the QR it complains about multiple outputs.

**Comment by cculianu** [comment]
ok, I was able to reproduce it.  You have to do it a few times. Looks like some sort of race condition that it doesn't happen every time. Working on it...

**Comment by cculianu** [comment]
Closed by 478444a08e95886064c34d555a1a001590f02f9c
